libr/main/rasm2.c

```diff
@@ -748,7 +751,9 @@ R_API int r_main_rasm2(int argc, const char *argv[]) {
 	}
 	R_FREE (log_level);
 	RAsmState *as = __as_new ();
-
+	if(!as){
+		return 0;
```

Comment:
Return 1 because thats an error
```suggestion
		return 1;
```